frontend/src/components/CardCreator/CardCreator.tsx

```diff
@@ -1,131 +1,444 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import "./CardCreator.css"
-import BlankCard from "../../Media/Cardimages/blank.png"
-import dark from "../../Media/Attributes/DARK.png"
-import light from "../../Media/Attributes/LIGHT.png"
-import divine from "../../Media/Attributes/DIVINE.png"
-import earth from "../../Media/Attributes/EARTH.png"
-import fire from "../../Media/Attributes/FIRE.png"
-import water from "../../Media/Attributes/WATER.png"
-import wind from "../../Media/Attributes/WIND.png"
-import spell from "../../Media/Attributes/SPELL.png"
-import trap from "../../Media/Attributes/TRAP.png"
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./CardCreator.css";
+
+import normalCard from "../../Media/Cardimages/common.png";
+import effectCard from "../../Media/Cardimages/effect.png";
+import fusionCard from "../../Media/Cardimages/fusion.png";
+import ritualCard from "../../Media/Cardimages/ritual.png";
+import synchroCard from "../../Media/Cardimages/synchro.png";
+import xyzCard from "../../Media/Cardimages/xyz.png"
+
+
+import dark from "../../Media/Attributes/DARK.png";
+import light from "../../Media/Attributes/LIGHT.png";
+import divine from "../../Media/Attributes/DIVINE.png";
+import earth from "../../Media/Attributes/EARTH.png";
+import fire from "../../Media/Attributes/FIRE.png";
+import water from "../../Media/Attributes/WATER.png";
+import wind from "../../Media/Attributes/WIND.png";
+
+
+import levelstar1 from "../../Media/Level/levelstar.png";
 
 function CardCreator() {
+  const [showedStars, setShowedStars] = useState(
+    <div className="star-container"></div>
+  );
+
+  const [showedRace, setShowedRace] = useState(<div></div>);
+  const [showedAttribute, setShowedAttribute] = useState(
+    <img src={dark} alt="" />
+  );
+  const [showedType, setShowedType] = useState(<div></div>);
+  const [showedCard, setShowedCard] = useState(<div className="card-container"><img className="card_img" src={effectCard} alt="" /></div>);
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('')
+  const [attack, setAttack] = useState("");
+  const [defence, setDefence] = useState("");
+
+
+  const changeDark = () => {
```

Comment:
Why do you have a different method for each attribute value? Just write a `changeAttribute` method that receives the value as parameter and use that parameter as src.

Reply:
I used a different type of method and its working :)